packages/viz/src/lib/Layer.ts

```diff
@@ -74,13 +77,22 @@ export class Layer {
    * Method to create the Deck.gl layer
    */
   public async _createDeckGLLayer() {
+    // The first step is to initialize the source to get the geometryType and the stats
+    const styleField = this._style ? this._style.field : undefined;
```

Comment:
:+1 Using one field is enough now, as our helpers require just one (maybe in the future we would use expressions with several ones, and then we'll add an array)